readme.md

```diff
@@ -205,6 +210,11 @@ Finally, just create `index.blade.php` and `show.blade.php` inside a `/views/blo
 
 Just run `php artisan canvas:setup` after installing Canvas. Then, navigate your browser to `http://your-app.test/blog` or any other URL that is assigned to your application. This command scaffolds a default frontend for your entire blog!
 
+If you want test data from the application, use the `--data` option:
+```bash
+php artisan canvas:setup --data
+```
+
```

Comment:
The `--data` option was removed from the project, so this can go.

Reply:
I implemented the --data option again to facilitate the development of the package and to allow users to quickly test the system. Do you believe that it is better not to implement this option?